assets/js/front/epoch-front-compiled.js

```diff
@@ -147,7 +147,7 @@ jQuery( document ).ready( function ( $ ) {
              *
              * @since 0.0.1
              */
-            $( app.form_el ).submit( function( event ) {
+            $( document ).on( 'submit', '#' + epoch_vars.form_id, function( event ) {
```

Comment:
Can I ask why switching to on() instead of submit() here? Also makes me nervous binding this to document, instead of the specific element, since we may be dealing with two different documents when in iFrame mode.